### DISCUSSION TITLE
unable to see books on Diskha portal for cisce,IGOT-health and kerala

### DISCUSSION THREAD

**Discussion by jankitrigyn** [Sunbird-Ed/Community#510] [Bugs] [answered]
unable to see books on diksha portal for CISCE, IGOT-Health and Kerala

**Comment by chitranshu-keshav** [comment]
@jankitrigyn Can you please provide some additional details to assist you. What exactly are you trying to do? What steps you have followed etc. so that some inputs can be provided

cc @rajeevsathish

**Reply by Krishnaj20** [reply]
@jankitrigyn - can you check if any books were published for these tenants or not? That could be the one of the possible reasons why no textbooks are available in these tenants

@kameshbhr

**Comment by jankitrigyn** [comment]
Hi Chiranshu Keshav,

Books are available in tanent wise but books are not visible in diksha portal.




Thanks & Regards,

Janki Kumar | Sr. Software Engineer

Phone: +91 7541054510

Email: ***@***.***

www.trigyn.com<http://www.trigyn.com/>





________________________________

**Reply by Krishnaj20** [reply]
Can you filter by collection do id and primary category instead of content?

**Comment by jankitrigyn** [comment]
From where can we  find do id.
________________________________

**Comment by rajeevsathish** [comment]
@jankitrigyn please refer this we are getting the textbook for igot-health
https://diksha.gov.in/explore?board=IGOT-Health&medium=Assamese&medium=Bengali&medium=English&medium=Gujarati&medium=Hindi&medium=Kannada&medium=Konkani&medium=Malayalam&medium=Marathi&medium=Others&medium=Punjabi&medium=Sign%20Language&medium=Tamil&medium=Telugu&medium=Urdu&gradeLevel=Nurses&gradeLevel=Police&gradeLevel=ANMs&gradeLevel=Others&gradeLevel=Allied%20%26%20Healthcare%20Professionals&gradeLevel=Dentist%20%26%20Dental%20Students&gradeLevel=Lab%20Technicians&gradeLevel=ASHA%20Volunteer&gradeLevel=NCC%20Cadets&gradeLevel=NSS%20Volunteers&gradeLevel=NYKS%20Volunteers&gradeLevel=Doctor%20(Allopathy)&gradeLevel=AYUSH%20Professional&gradeLevel=Pharmacist&gradeLevel=Psychosocial%20HealthProf&gradeLevel=Rehabilitation%20Prof.&gradeLevel=Student%2F%20Intern&gradeLevel=Other%20Health%20Worker&gradeLevel=Volunteer%20Red%20Cross&gradeLevel=Volunteer%20BSG&gradeLevel=Other%20Volunteer&gradeLevel=NGO%20Volunteer&gradeLevel=Sanitation%20Prof.&gradeLevel=District%20Administrator&gradeLevel=RWA%20representative&gradeLevel=Common%20citizen&gradeLevel=TB%20Champions&id=igot_health&&selectedTab=textbook


<img width="1723" alt="Screenshot 2023-06-05 at 4 16 59 PM" src="https://github.com/Sunbird-Ed/Community/assets/5653717/1f019afb-9ecd-4e9f-b727-1a25d37c96bd">

and also getting textbook for kerala board also 
https://diksha.gov.in/explore?board=State%20(Kerala)&medium=Arabic&medium=English&medium=Hindi&medium=Kannada&medium=Malayalam&medium=Sanskrit&medium=Tamil&medium=Urdu&gradeLevel=Class%201&gradeLevel=Class%202&gradeLevel=Class%203&gradeLevel=Class%204&gradeLevel=Class%205&gradeLevel=Class%206&gradeLevel=Class%207&gradeLevel=Class%208&gradeLevel=Class%209&gradeLevel=Class%2010&gradeLevel=Class%2011&gradeLevel=Class%2012&gradeLevel=Others&gradeLevel=CPD&id=kl_k-12&&selectedTab=textbook

<img width="1721" alt="Screenshot 2023-06-05 at 4 17 10 PM" src="https://github.com/Sunbird-Ed/Community/assets/5653717/51f1d54a-c7ca-4821-a08b-ae0713712ed9">

**Comment by jankitrigyn** [comment]
Hi,
So I have to select all checkbox, but what about cisce digital textbook is not coming but when we are selecting all then it is coming.



Thanks & Regards
Janki Kumar
________________________________

**Reply by rajeevsathish** [reply]
@jankitrigyn for cisce there is no textbook created so it is not coming

**Comment by jankitrigyn** [comment]
Hi,
I am attaching screen short of CISCE books are available.


Thanks & Regards
Janki Kumar





________________________________

**Comment by chitranshu-keshav** [comment]
@jankitrigyn Can this issue be closed?

**Comment by ashokreddy1208** [comment]
@jankitrigyn Can you please mark this thread as answered?

**Reply by Krishnaj20** [reply, marked as answer]
@jankitrigyn - closing this as we haven't received any response from the requestor. Feel free to start a new thread if you have any follow up questions. Thanks

**Comment by jankitrigyn** [comment]
Thanks



Thanks & Regards
Janki kumar.
________________________________
